lib/model/CModelFactory.cc

```diff
@@ -280,6 +285,10 @@ void CModelFactory::swap(CModelFactory& other) {
     m_InfluenceCalculatorCache.swap(other.m_InfluenceCalculatorCache);
 }
 
+CModelFactory::TInterimBucketCorrectorPtr CModelFactory::interimBucketCorrector() const {
+    return TInterimBucketCorrectorPtr{m_InterimBucketCorrector};
```

Comment:
throws if m_InterimBucketCorrector (weak ptr) is gone

Reply:
As below, but maybe at least warrants a comment.